proposals/zips/zip001.sol

```diff
@@ -19,12 +19,12 @@ import {CoreRef} from "@protocol/refs/CoreRef.sol";
 
 contract zip001 is Proposal, TimelockProposal {
     string public name = "ZIP001";
-    string public description = "The ZTX Base System proposal";
+    string public description = "The ZTX Base System proposal for the wearable drop";
```

Comment:
Being overly pedantic, but It's for the Halloween wearables drop 😆 

Additionally, won't we be using this same contract for future wearables too? I think it should simply be "The ZTX Wearables proposal", personally.